package/Resources/Translations/zh-Hans/Translation.ini

```diff
@@ -0,0 +1,1707 @@
+INI:Hotkeys:CenterBase:Description=将视角中心对准你的基地; Center the camera on your base.
+INI:Hotkeys:CenterBase:UIName=居中视野到基地; Center Base
+INI:Hotkeys:CenterOnRadarEvent:Description=将视角对准雷达最新报告的事件; Center the camera around the latest radar event.
+INI:Hotkeys:CenterOnRadarEvent:UIName=转到雷达事件; Goto Radar Event
+INI:Hotkeys:CenterView:Description=将视角中心对准所选对象; Center the camera to the selected objects.
+INI:Hotkeys:CenterView:UIName=居中视野; Center View
+INI:Hotkeys:CombatantSelect:Description=选中所有战斗单位; Select units considered as combatants.
+INI:Hotkeys:CombatantSelect:UIName=选择战斗单位; Combatant Select
+INI:Hotkeys:DefenseTab:Description=右侧栏位跳转至防御支援页面; Jump to sidebar armory tab.
+INI:Hotkeys:DefenseTab:UIName=防御支援页面; Armory Tab
+INI:Hotkeys:Delete:Description=删除选定的通讯信标; Delete a selected map beacon.
+INI:Hotkeys:Delete:UIName=删除信标; Delete Beacon
+INI:Hotkeys:DeployObject:Description=部署所选单位; Deploy selected units.
+INI:Hotkeys:DeployObject:UIName=部署; Deploy Object
+INI:Hotkeys:Follow:Description=让视野跟随所选对象; Make the selected objects follow another object.
+INI:Hotkeys:Follow:UIName=视野跟随; Follow
+INI:Hotkeys:GuardObject:Description=使收到该指令的单位守卫附近的区域, 自动攻击靠近的敌人; Make your selected units guard the nearby area and automatically attack enemies.
+INI:Hotkeys:GuardObject:UIName=警戒; Guard
+INI:Hotkeys:HealthNav:Description=按生命值降序选择作战单位; Select units based on current health.
+INI:Hotkeys:HealthNav:UIName=按生命值顺序选择; Health Navigation
+INI:Hotkeys:InfantryTab:Description=右侧栏位跳转至步兵页面; Jump to sidebar infantry tab.
+INI:Hotkeys:InfantryTab:UIName=步兵页面; Infantry Tab
+INI:Hotkeys:NextObject:Description=选择下一个单位; Select the next unit.
+INI:Hotkeys:NextObject:UIName=下一个单位; Next Unit
+INI:Hotkeys:Options:Description=打开游戏内的选项菜单; Open the in-game Options menu.
+INI:Hotkeys:Options:UIName=选项菜单; Options Menu
+INI:Hotkeys:PlaceBeacon:Description=在地图上放置一个通讯信标来提醒你的队友; Place a beacon on the map to alert your teammates.
+INI:Hotkeys:PlaceBeacon:UIName=通讯信标; Place Beacon
+INI:Hotkeys:PlanningMode:Description=使用路径点模式制定单位行动路线; Enable waypoint mode.
+INI:Hotkeys:PlanningMode:UIName=路径点模式; Waypoint Mode
+INI:Hotkeys:PreviousObject:Description=选择上一个单位; Select the previous unit.
+INI:Hotkeys:PreviousObject:UIName=上一个单位; Previous Unit
+INI:Hotkeys:ScatterObject:Description=使被你选中的单位分散开; Make your selected units scatter.
+INI:Hotkeys:ScatterObject:UIName=散开; Scatter
+INI:Hotkeys:ScreenCapture:Description=捕获当前游戏画面并保存至游戏安装目录; Takes a screenshot and saves it in your game directory.
+INI:Hotkeys:ScreenCapture:UIName=截图; Screen Capture
+INI:Hotkeys:SetView1:Description=创建书签1; Sets bookmark 1.
+INI:Hotkeys:SetView1:UIName=设置书签 1; Set Bookmark 1
+INI:Hotkeys:SetView2:Description=创建书签2; Sets bookmark 2.
+INI:Hotkeys:SetView2:UIName=设置书签 2; Set Bookmark 2
+INI:Hotkeys:SetView3:Description=创建书签3; Sets bookmark 3.
+INI:Hotkeys:SetView3:UIName=设置书签 3; Set Bookmark 3
+INI:Hotkeys:SetView4:Description=设置书签 4; Sets bookmark 4.
+INI:Hotkeys:SetView4:UIName=创建书签4; Set Bookmark 4
```

Comment:
Please keep consistency with the space character between a Chinese character and an alphabet/digit. Add the missing space character for the description.